Server/Controllers/MatchController.cs

```diff
@@ -341,6 +361,7 @@ public class BeforeMatchRequest
     public class BeforeMatchResponse
     {
         public IEnumerable<Player> Players { get; set; }
+        public IEnumerable<LeaderBoardPlayer> LeaderBoard { get; set; }
```

Comment:
```suggestion
        public IEnumerable<LeaderboardPlayer> Leaderboard { get; set; }
```

Reply:
![image](https://user-images.githubusercontent.com/16607844/79550507-16460900-80a1-11ea-82ac-d6283bc9924b.png)
resharper suggests this is a typo. I've changed it anyway